src/cdk/overlay/fullscreen-overlay-container.spec.ts

```diff
@@ -111,7 +111,7 @@ describe('FullscreenOverlayContainer', () => {
   providers: [Overlay],
 })
 class TestComponentWithTemplatePortals {
-  @ViewChild(CdkPortal) templatePortal: CdkPortal;
+  @ViewChild(CdkPortal, {static: true}) templatePortal: CdkPortal;
```

Comment:
Maybe we should put in `detectChanges` calls instead? My guess here is that the first test used static queries and then we ended up copying it everywhere.

Reply:
I decided to use `static` for this one since it keeps the test as running just one change detection, even though it's largely trivial.